static/src/questionnaires/QuestionnaireCreate.vue

```diff
@@ -355,28 +351,33 @@ export default Vue.extend({
       }
       return saveMethod(this.currentQuestionnaire)
     },
+    validateCurrentForm() {
```

Comment:
You could refactor this part as follow: 
`states = {
  STATES.PREVIEW: true,
  STATES.START: this.$refs.questionnaireMetadataCreate.validateForm()
...
}
validateCurrentForm() {
  return state[this.state]
}
`